types/stripe-v3/stripe-v3-tests.ts

```diff
@@ -129,6 +129,22 @@ describe("Stripe elements", () => {
         card.destroy();
     });
 
+    it("should create an iban source wit a mandate", () => {
+        stripe.createSource({
```

Comment:
Maybe construct the SourceOptions type as above? https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41924/files#diff-9259bd018b02a0dd1a4bb61db566ec1eR81?

Reply:
I gotta say, I'm not seeing how it's improving readability or anything

Reply:
I only meant for the sake of consistency with the existing tests.

Reply:
Since there was a typo to fix, I changed it anyway. Is this what you meant ?

Reply:
I think I actually just misread that - I thought the OwnerInfo type was being used but it's actually just an assignment... sorry about that.